### PR TITLE
osm2pgsql 0.90.1

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by DomT4** [comment]
Looks like they've gone over to `cmake`.

**Comment by ilovezfs** [comment]
@DomT4 PR refreshed.

File: Formula/osm2pgsql.rb [hunk 1]
```diff
@@ -1,51 +1,43 @@
...
-      "--with-proj=#{Formula["proj"].opt_prefix}",
-      "--with-boost=#{Formula["boost"].opt_prefix}",
-      "--with-zlib=/usr",
-      "--with-bzip2=/usr",
-    ]
-    puts args
-    if build.with? "protobuf-c"
-      args << "--with-protobuf-c=#{Formula["protobuf-c"].opt_prefix}"
+    args = std_cmake_args
+    args += ["-DWITH_LUA=OFF"] if build.without? "lua"
```
Comment (DomT4) [inline Formula/osm2pgsql.rb]: Why not just `args << "-DWITH_LUA=OFF" if build.without? "lua"` ?

Comment (ilovezfs) [inline Formula/osm2pgsql.rb]: okie dokie

Comment (ilovezfs) [inline Formula/osm2pgsql.rb]: fixed

**Comment by DomT4** [comment]
👍 when 🍏

**Pull request closed without merging**